zjit/src/codegen.rs

```diff
@@ -379,6 +381,53 @@ fn gen_putspecialobject(asm: &mut Assembler, value_type: SpecialObjectType) -> O
     )
 }
 
+fn gen_getspecial_symbol(asm: &mut Assembler, _key: u64, svar: u64) -> Opnd {
```

Comment:
I think you need to save the PC for some of these because they can allocate or raise